### PR TITLE
0315 - Chore: Change Quickstart countdown timer to 10 minutes

### PR DESCRIPTION
**Pull request by robbiecarlton**
This changes the timer to 10 minutes, and adds a message when the timer runs to zero.

Pivotal ticket: https://www.pivotaltracker.com/story/show/170900315

### CODE REVIEW

File: status/res/index.js [hunk 1]
```diff
@@ -6,29 +6,42 @@ const agentUrl = `https://${authData.holochain_agent_id}.holohost.net`
 
 const redirectIfOnline = async url => {
   try {
-    const res = await fetch(url, { mode: 'no-cors' })
+    await fetch(url, { mode: 'no-cors' })
```
Comment (yegortimoshenko) [inline status/res/index.js]: :+1:

File: status/res/index.js [hunk 2]
```diff
@@ -6,29 +6,42 @@ const agentUrl = `https://${authData.holochain_agent_id}.holohost.net`
...
+  $minutes.textContent = clockPad(minutes)
+  $seconds.textContent = clockPad(seconds)
+  if (minutes === 0 && seconds === 0) {
+    $stepBox.innerHTML = `
+      <div class="stepLabel step5">
+        <p class="subheader-text-bookend text-white">
+          Hmm. This is taking longer than normal. Please leave this window open for another few minutes and check back later. 
+          If you're still not re-directed to your HoloPort admin panel by then, please contact <a href="https://forum.holo.host" target="_blank" class="text-white underline">customer service</a>.
+        </p>
+      </div>
```
Comment (yegortimoshenko) [inline status/res/index.js]: This should be in `index.html`.

File: status/res/index.js [hunk 3]
```diff
@@ -6,29 +6,37 @@ const agentUrl = `https://${authData.holochain_agent_id}.holohost.net`
...
   const delta = new Date(Math.max(0, estimate - Date.now()))
-  $minutes.textContent = clockPad(delta.getMinutes())
-  $seconds.textContent = clockPad(delta.getSeconds())
+  const minutes = delta.getMinutes()
+  const seconds = delta.getSeconds()
+  $minutes.textContent = clockPad(minutes)
+  $seconds.textContent = clockPad(seconds)
+  if (minutes === 0 && seconds === 0) {
+    $stepBoxDefault.style.display = 'none'
+    $stepBoxTimeUp.style.display = 'flex'
```
Comment (robbiecarlton) [inline status/res/index.js]: @yegortimoshenko like this?

Comment (yegortimoshenko) [reply status/res/index.js]: Much better than 82c5d12adf2264abbbf6d8735b6dd6747ec5e5ba, but it now leaks CSS semantics into JS. See suggested alternative in #6, specifically 6635efe263c29d87948ae95214ade20d544e0e0b.

**Comment by robbiecarlton** [comment]
Cool. I merged this and will leave it your call whether you want to merge #6

**Pull request merged**